person.rb

```diff
@@ -1,6 +1,6 @@
 require './nameable'
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
- [ ] Retels should have `Getters and Setters` since we'll be making modifications by class method